### Introduction to RNA-seq with Galaxy

## Description
This course is an introduction to RNA sequencing using Galaxy on [Tufts University's Galaxy server](https://galaxy.cluster.tufts.edu). 
For support with this workshop or Galaxy, see our [Galaxy documentation](https://it.tufts.edu/research-technology/bioinformatics/tufts-galaxy) or email TTS Research Technology [tts-research@tufts.edu](mailto:ltts-research@tufts.edu)


## Goals

Bioinformatics for RNAseq has the following workflow steps

<img src="img/workflow.png" width="200">

### Materials Needed
- Chrome web browser
- Account on [Tufts High Performance Compute Cluster](https://it.tufts.edu/research-technology/high-performance-computing)
- [VPN](https://access.tufts.edu/vpn) if accessing the HPC from off campus

## Table of Contents
- [Introduction Slides](slides/galaxyWorkshop_idgh1001_15Feb2022.pdf)
- [Introduction to Galaxy](lessons/00_Galaxy_introduction.md)
- [Next: Process Raw Reads](lessons/01_Introduction_and_Setup_noqual.md)
- [Read Alignment](lessons/03_Read_alignment.md)
- [Gene quantification](lessons/04_Gene_quantification.md)
- [Differential Expression](lessons/05_Diff_expression.md)


## Acknowledgement
Thanks to bioinformatics intern [Meg Farely](https://www.linkedin.com/in/meg-farley-93b311160/) for help developing this workshop.
Much of this explanation has been adapted from these sources:
- [Harvard Chan Bioinformatics Core RNAseq training](https://hbctraining.github.io/DGE_workshop)
- [DESeq2 Vignette](http://www.bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html)
- [Introduction To RNA-seq with Galaxy](https://rbatorsky.github.io/intro-to-rnaseq-with-galaxy/)


